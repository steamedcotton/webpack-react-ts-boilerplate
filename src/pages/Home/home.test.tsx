import React from 'react';
import { Home } from './';
import renderer from 'react-test-renderer';

test('<Home /> Renders', () => {
    const component = renderer.create(<Home />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
