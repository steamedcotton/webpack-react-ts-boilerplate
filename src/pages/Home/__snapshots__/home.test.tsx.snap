// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> Renders 1`] = `
<div
  className="sc-bdfBwQ iQqEuM"
>
  <h1>
    React App
  </h1>
  <h2>
    Website
  </h2>
</div>
`;
